Sources/Core/AWSSDKIdentity/Tests/AWSSDKIdentityTests/AWSCredentialIdentityResolverTests/ProcessAWSCredentialIdentityResolverTests.swift

```diff
@@ -8,11 +8,28 @@
 import XCTest
 import struct AWSSDKIdentity.ProcessAWSCredentialIdentityResolver
 @_spi(FileBasedConfig) @testable import AWSClientRuntime
+import struct AWSSDKIdentity.ProfileAWSCredentialIdentityResolver
 
 class ProcessAWSCredentialIdentityResolverTests: XCTestCase {
     let configPath = Bundle.module.path(forResource: "config_with_process", ofType: nil)!
     let credentialsPath = Bundle.module.path(forResource: "credentials", ofType: nil)!
 
+    // Importing & instantiating ProfileAWSCredentialIdentityResolver in setUp() is a temporary workaround for
+    //  CRT's ProcessAWSCredentialIdentityResolver missing a required setup and depending on
+    //  some other credential resolver for it. The bug was caught when ECSAWSCredentialIdentityResolver
+    //  unit tests were refactored to use SDK-side implementation.
+    //  Xcode runs unit tests alphabetically, and the 2 tests below always ran after ECS resolver's unit tests.
+    //  Removing CRT's ECS resolver instantiation from the ECS resolver tests revealed the bug & started failing 2 tests below.
+    override class func setUp() {
+        let configPath = Bundle.module.path(forResource: "config", ofType: nil)!
+        let credentialsPath = Bundle.module.path(forResource: "credentials", ofType: nil)!
+        super.setUp()
+        _ = try! ProfileAWSCredentialIdentityResolver(
+            configFilePath: configPath,
+            credentialsFilePath: credentialsPath
+        )
```

Comment:
Configuring the file paths is okay but not needed.
Merely configuring the `ProfileAWSCredentialIdentityResolver` with default paths and having it fail to resolve those paths is sufficient for making the process credential provider tests pass.